README.md

```diff
@@ -48,9 +48,9 @@ import * as PXBThemes from '@pxblue/react-native-themes';
 
 ```tsx
 import { Provider as ThemeProvider} from 'react-native-paper';
-import * as PXBThemes from '@pxblue/react-native-themes';
+import * as Themes from '@brightlayer-ui/react-native-themes';
```

Comment:
We need to BLUIThemes instead of Themes since using this at line 53. Can you please recheck ?

Reply:
Done.